3.6/architecture-deployment-modes-cluster-sharding.md

```diff
@@ -24,6 +24,8 @@ data, which would not fit on a single machine. A high rate of write / read
 operations or AQL queries can also overwhelm a single servers RAM and disk
 capacity.
 
+ArangoDB is also available as [cloud service - **ArangoDB Oasis**](https://cloud.arangodb.com/home?utm_source=docs&utm_medium=cluster_pages&utm_campaign=docs_traffic){:target="_blank"}. Fire up your cluster in just a few clicks with the [14-day free trial](https://cloud.arangodb.com/home?utm_source=docs&utm_medium=cluster_pages&utm_campaign=docs_traffic){:target="_blank"}.
```

Comment:
Too disruptive. It needs to go or say something about how easy it is to scale with Oasis - but that's dangerous territory. We do not support resharding (only balancing)!

Reply:
OK. Let's remove it.

Reply:
removed